### PR TITLE
book: remove a sentence that is not clear

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/doc/trpl/generics.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 % Generics
 
 Sometimes, when writing a function or data type, we may want it to work for
-multiple types of arguments. Luckily, Rust has a feature that gives us a better
-way: generics. Generics are called ‘parametric polymorphism’ in type theory,
+multiple types of arguments. In Rust, this feature is called generics.
```
Comment (steveklabnik) [inline src/doc/trpl/generics.md]: What about "We can do this in Rust with generics."?

Comment (tshepang) [inline src/doc/trpl/generics.md]: Nice one.

Done, with a readability tweak.

**Pull request merged**